server/routes/assessment.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+
+export default function createRouter(assessment) {
+  const router = express.Router();
+
+  router.post('/', (req, res) => {
+    assessment.record(req.body)
+      .then(
+        result => res.json({
+          status: 'OK',
+          data: { id: result.assessment_id },
+        }),
+        (err) => {
+          // TODO: proper logging lib for server stuff?
+          // eslint-disable-next-line no-console
+          console.warn('Failed to save assessment', err.stack);
```

Comment:
I tend to use error = crashed app, warn = failed request.

But it makes no different in this scenario as console.warn and console.error both just write to stderr.